src/js/comments.js

```diff
@@ -24,26 +24,37 @@ class Comments {
 	 */
 	_renderComments () {
 		/*global Coral*/
-		const scriptElement = document.createElement('script');
-		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
-		scriptElement.onload = () => Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'),
-			{
-				talk: 'https://ft-next-talk-spike.herokuapp.com',
-				events: (events) => {
-					events.onAny((name, data) => {
-						const message = new CustomEvent('talkEvent', {
-							detail: {
-								name,
-								data
-							}
-						});
-						parent.dispatchEvent(message);
-					});
-				}
-			}
-		);
-
-		this.oCommentsEl.appendChild(scriptElement);
+		fetch(`https://comments-auth.ft.com/v1/jwt/`, {
```

Comment:
This is fine in here for now, in the long run whilst we make sure everything works correctly but in the long run I would prefer this to live in its own file/module to keep it tidy.